packages/now-build-utils/src/fs/run-user-scripts.ts

```diff
@@ -291,7 +291,7 @@ export async function runNpmInstall(
   } else {
     opts.prettyCommand = 'yarn install';
     command = 'yarn';
-    commandArgs = args.concat(['install', '--ignore-engines']);
+    commandArgs = ['install', ...args, '--ignore-engines'];
```

Comment:
Thanks! Please add a test fixture under `now-build-utils/test/fixtures` so we can see what this is fixing.

Reply:
So from what I understand I have to add a sample svelte project there right?

Reply:
Yes, you would add a project that would fail with the current behavior but is fixed with your change to the arguments.

I'm guessing you have yarn 2 as a dev dependency?

The original description doesn't explain how to reproduce the error or what the error says.

Reply:
Update: I just saw #4436.

You can add the code from your repo as a test fixture

Reply:
@styfle Yarn 2 works in an interesting way.

The version that's installed is actually yarn 1.22 (usually this is installed globally not as a dev dependency), but you set it to use Yarn 2 in the project level by running the command `yarn set version berry` which then sets up that project to use the latest yarn version.

I think it should still be reproducible in a sample project I think, due to yarn installing Yarn 2 in the .yarn folder.

Reply:
Yes @MarkLyck is kinda right, globally any version can be installed, but then you can also have an another local version installed in the `.yarn` folder and point to it through the `.yarnrc` (in case of v2 `.yarnrc.yml`) file. This behavior is not specific to yarn v2, you can try doing `yarn set version latest` in an non v2 project too.

Thus, generally, projects using v2 have it installed locally, with the help of `yarn set version berry`.

And Alright will be spinning up the default demo project with yarn v2!

Reply:
@styfle Added a Template Svelte project with yarn v2

And @MarkLyck The behavior of pointing to local yarn version is not specific to yarn v2. Even on yarn v1 you could point the global yarn to use a local yarn version. At moment using yarn v2 globally is not suggested, thus it is generally downloaded locally.